README.md

```diff
@@ -33,6 +33,9 @@ To use this image you must do as follows:
 # you can use tags latest
 docker pull chentex/random-logger:latest
 
+# use different intervals to print logs every random(100, 400) milliseconds
+docker pull chentex/random-logger:latest 100 400
```

Comment:
Thanks for your contribution, I think this must be `docker run` instead of `docker pull`